docs/user/README.md

```diff
@@ -18,22 +18,66 @@ By default, the cluster will be given the name `kind-1`. "1," here, is the
 default context name. 
 Use the `--name` flag to assign the cluster a different context name.
 
-Once a cluster is created you can use [kubectl][kubectl] to interact with it by
-using the configuration file generated by `kind`:
+`kind` also allows you to customize your Kubernetes cluster by using a 
```

Comment:
this should probably be in the advanced docs for now, nothing in here should be touched by a typical user. I want to make it possible to get started with `kind` with the minimum of information, with details covering complex usage saved for later in the docs.

Reply:
Moved that to its own section. What do you think?